pym/bob/scm/scm.py

```diff
@@ -35,6 +36,7 @@ def __setstate__(self, s):
         self.__init()
 
     def __doesMatch(self, scm, env):
+        if self.__if and not env.evaluate(self.__if, "scmOverride::if"): return False
```

Comment:
Shouldn't that be

```suggestion
        if self.__if is not None and not env.evaluate(self.__if, "scmOverride::if"): return False
```

? Because an empty string should be considered `False`.